scale/product/urls.py

```diff
@@ -1,12 +1,4 @@
 """Defines the URLs for the RESTful product services"""
 from django.conf.urls import url
 
-import product.views as views
-
-urlpatterns = [
-    url(r'^products/$', views.ProductsView.as_view(), name='products_view'),
-    url(r'^products/updates/$', views.ProductUpdatesView.as_view(), name='product_updates_view'),
-    url(r'^products/(?P<product_id>\d+)/$', views.ProductDetailsView.as_view(), name='product_details_view'),
-    url(r'^products/(?P<file_name>[\w.-]{0,250})/$', views.ProductDetailsView.as_view(), name='product_details_view'),
-    url(r'^products/(?P<product_id>\d+)/sources/$', views.ProductSourcesView.as_view(), name='product_sources_view'),
-]
+urlpatterns = []
```

Comment:
Either we should delete the urls.py or leave a comment in the file explaining the lack of patterns.

Reply:
Removed the urls.py file